observer/bin/spark-observer.js

```diff
@@ -47,7 +43,14 @@ const loop = async (name, fn, interval) => {
     }
     const dt = Date.now() - start
     console.log(`Loop "${name}" took ${dt}ms`)
-    await timers.setTimeout(interval - dt)
+    recordTelemetry(`loop_${name.replaceAll(' ', '_')}`, point => {
+      point.intField('interval_ms', interval)
+      point.intField('duration_ms', dt)
+      point.intField('delay_ms', interval - dt)
+    })
+    if (dt < interval) {
```

Comment:
This condition isn't necessary as `setTimeout(-10)` has the same result as `setTimeout(0)`. Up to you

Reply:
I was not aware of that. I replaced the condition with a code comment explaining this Node.js feature.

Reply:
It's not a Node.js feature apparently, it's part of the spec: https://stackoverflow.com/a/8431015

I find the comment worse than the condition (it makes me think more). Would you be ok with reverting to the condition?

Reply:
Sure, no problem, I'll use the condition.